index.js

```diff
@@ -93,7 +96,25 @@ const defaults = userOptions => {
   if (!options.include || !options.include.length) {
     console.log("🔥  include option should be an non-empty array");
     exit = true;
+  } else {
```

Comment:
This is wrong. We should be able to specify paths to be included as well as paths to be excluded.
Let's say I want to include landing page that is not linked from anywhere but want to exclude login and register patsh.